lib/exercism/rikki.rb

```diff
@@ -1,19 +1,27 @@
 require 'digest/sha1'
 
-class Rikki
-  def self.validate(key)
+module Rikki
+  extend self
```

Comment:
>I really don't like this change.
It obscures that these are all class methods.
I'd like to see the self. prefixes back.

Understood. Just thought making them module methods DRY'd it up a bit especially since we're not instantiating anything or taking advantage of `class` features.

> Although, since it is a module perhaps module_function would be more appropriate:
https://github.com/bbatsov/ruby-style-guide#module-function

Glad you pointed that out. I wonder why `extend self` is preferred over `module_function`. Ah well, if we stay the module route vs class, I'll change to this.

> I'm not deep into Rikki so I may be wrong here.
But I think that Rikki should still be a class, since it is likely that there will be more than one of them?

I think rikki is singular but I'll defer to @kytrinyx  for that answer as I'm not sure (merely adding more languages to he/she/it's vocabulary).